lessons/basics/comprehensions.md

```diff
@@ -105,7 +105,7 @@ iex> for {k, v} <- [one: 1, two: 2, three: 3], into: %{}, do: {k, v}
 %{one: 1, three: 3, two: 2}
 ```
 
-Since bitstrings are enumerable we can use list comprehensions and `:into` to create strings:
+Since binaries are collectable we can use list comprehensions and `:into` to create strings:
```

Comment:
Is "collectables" more appropriate here than "enumerable"?  I can't say I've ever read "X is collectable", it's typically some derivative of "enumerable".